docs/interactive/tardis_example.yml

```diff
@@ -0,0 +1,57 @@
+# Example YAML configuration for TARDIS
```

Comment:
What's the purpose of this file as per you?

Reply:
I figured the user won't have to download the yml file separately. Should I remove it?

Reply:
Yes you can if they don't concern with the work you did in notebook.